bitnami/mysql/Chart.yaml

```
@@ -25,4 +25,4 @@ name: mysql
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/mysql
   - https://mysql.com
-version: 9.7.1
+version: 9.7.2
```

As this is a new addition this should be a minor version bump.

```suggestion
version: 9.8.0
```